index.html

```diff
@@ -1,221 +1,213 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Sajjda Portfolio</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
 
-  <body>
-    <header id="main-header">
-      <div class="toolbarContainer">
-        <h2 id="logo">Sajjad</h2>
-        <nav>
```

Comment:
Kindly put the logo of your project inside an anchor (<a>) tag, so that it will be cliquable.